index.md

```diff
@@ -1,131 +1,198 @@
+This document doesn’t pretend to solve anything but focus people on our common needs as a species. If 
+people believe in this manifesto, we encourage people to take this manifesto and adapt it for your region 
+of the world to address your specific situation. And then we must force our governments to uphold this 
+manifesto to take decisive action to improve the lives of everyone. If the governments refuse, then it 
+is a clear indication that they no longer represent the people and more drastic change is required.
 
-The precise next steps are up to each country and region to decide. We encourage you to help spread this manifesto, to help translate it and to build derivative versions specific to your country or region. And then step into action!
+The precise next steps are up to each country and region to decide. We encourage you to help spread this 
+manifesto, to help translate it and to build derivative versions specific to your country or region. And 
+then step into action!
 
-The remainder of this document is a more in depth explanation of the points in the manifesto. It goes a little deeper into defining the problems and outlining our thinking.
+The remainder of this document is a more in depth explanation of the points in the manifesto. It goes a 
+little deeper into defining the problems and outlining our thinking.
 
 
 Bail out people, not corporations
 ---------------------------------
 
-Events like pandemics where the economy of a country crashes, require the government to step in and help to restart the economy. In the recent past many governments have chosen to invest in companies in an effort to stimulate the economy with the theory that money would eventually reach the people.
+Events like pandemics where the economy of a country crashes, require the government to step in and help to 
+restart the economy. In the recent past many governments have chosen to invest in companies in an effort to 
+stimulate the economy with the theory that money would eventually reach the people.
 
-Most of the time that never happened and during the Coronavirus pandemic, we are seeing airlines pay massive dividends at the same time as they are being bailed out. If we bail out corporations, we’re only going to increase the divide between the rich and poor, leading to greater desperation and poverty which will drive up crime.  
+Most of the time that never happened and during the Coronavirus pandemic, we are seeing airlines pay massive 
+dividends at the same time as they are being bailed out. If we bail out corporations, we’re only going to 
```

Comment:
Also we see money going to companies well known to not being acting to preserve environment and contributing to global warming. Paid dividends aren't the sole issue with this approach.

Reply:
I see your point, but I think it gets the point across as is.